src/main/java/org/openlmis/buq/service/buq/BottomUpQuantificationService.java

```diff
@@ -78,6 +81,7 @@
 @Service
 public class BottomUpQuantificationService {
 
+  public static final String APPROVE_BUQ_RIGHT_NAME = "APPROVE_BUQ";
```

Comment:
Nitpick: usually you would have an empty line between 84 and 85.

Reply:
I added empty line below the constant.